arches_lingo/src/arches_lingo/components/generic/MetaStringViewer.vue

```diff
@@ -160,7 +160,7 @@ async function deleteSectionValue(tileId: string) {
             </template>
         </DataTable>
     </div>
-    <div v-else>{{ props.metaStringText.noRecords }}</div>
+    <p v-else>{{ props.metaStringText.noRecords }}</p>
```

Comment:
nit: I tend to shy away from `p` tags because of the inherit padding. However if it's desired then you do you 👍

Reply:
Understood. I made this change to bring this in line with other elements elsewhere in the report that are rendered this way. e.g. https://github.com/archesproject/arches-lingo/blob/3e3e1760a22b8992f0dd8ec1e6d0c71c2991b206/arches_lingo/src/arches_lingo/components/scheme/SchemeLicense/components/SchemeLicenseViewer.vue#L98

Reply:
fair -- happy to leave for now